File: DashboardClient/TBoardRabbitAdapter/src/main/java/broker/messages/incoming/ServiceEvent.java
package broker.messages.incoming;

import java.util.Date;

public class ServiceEvent {

	public String sourceId;

	public ServiceType sourceType;

	public Date time;

	public String customMessage;

	public ServiceEvent(
			String sourceId,
			ServiceType type,
			String customMessage) {

		this.sourceId = sourceId;
		this.time = new Date();
		this.sourceType = type;
		this.customMessage = customMessage;
	}
}

File: Observers/DataObserver/src/broker/service-event.ts
import ServiceType from './service-type'

export default class ServiceEvent {
	sourceId: String
	sourceType: ServiceType;
	time: Date;
	customMessage: String

	constructor(
		sourceId: String,
		sourceType: ServiceType,
		time: Date,
		customMessage: String) {

		this.sourceId = sourceId;
		this.sourceType = sourceType;
		this.time = time;
		this.customMessage = customMessage;
	}

}
